<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>1161</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1161</width>
       <height>478</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Gestion Client</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>1091</width>
        <height>431</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter Client</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id_Client</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>130</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>tel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tel">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mail">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>280</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>100</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Poids</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>160</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Taille</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>30</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>230</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_4">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>30</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Homme</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Femme</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_5">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>90</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>50-60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60-70</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>70-80</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80-90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90-100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100-110</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>110-120</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>90</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Kg</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_6">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>150</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>150-160</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>160-170</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>170-180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>180-190</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>190-200</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>150</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>cm</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_7">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>230</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>18-25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26-40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>41-60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>plus que 60</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>230</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ans</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter1">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>350</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher Client</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>841</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier1">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Chercher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Id</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prenom</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Trier</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Id</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prenom</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_supp1">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>320</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1161</width>
       <height>478</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Gestion Commandes de Ventes</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>1111</width>
        <height>451</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajouter Commandes de Ventes</string>
       </attribute>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id_CommandesV</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom_Produit</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>250</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>340</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_Fabrication</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>40</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>120</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>250</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="l9">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>340</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Modifier Commandes de Ventes</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>271</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id_CommandesV</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>870</x>
          <y>170</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>280</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom_Produit</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>130</y>
          <width>271</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>240</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>240</y>
          <width>271</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>330</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_Fabrication</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>320</y>
          <width>271</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Afficher Commandes de Ventes</string>
       </attribute>
       <widget class="QTableView" name="tableView_Com">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>871</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_8">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Rechercher</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Id_CommandesV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nom_Produit</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_9">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Trier</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Id_Commandesv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nom_Produit</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pushButton_modif2">
        <property name="geometry">
         <rect>
          <x>990</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supp2">
        <property name="geometry">
         <rect>
          <x>990</x>
          <y>50</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer Doc</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>980</x>
          <y>360</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Imprimer PDF</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Supprimer Commandes de Ventes</string>
       </attribute>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id_CommandesV</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>180</y>
          <width>281</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1161</width>
       <height>478</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Statistiques</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
